fabric-gametest-api-v1/src/main/java/net/fabricmc/fabric/api/gametest/v1/CustomTestMethodInvoker.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.gametest.v1;
+
+import java.lang.reflect.Method;
+
+import net.minecraft.test.TestContext;
+
+/**
+ * Implement this interface on test suites to provide custom logic for invoking {@link GameTest} test methods.
+ */
+public interface CustomTestMethodInvoker {
```

Comment:
Could maybe add a deprecated `FabricGameTest` interface that subclasses this interface, so that people can easily find the right migration ?

Reply:
Humm, im not sure how useful that would be. This module was basically rewritten from scratch, I dont think we should aim to add new deprecated code just to aid with porting. Chances are we would need to carry it around for years if we did.